core/src/autogluon/core/models/abstract/abstract_model.py

```diff
@@ -1606,8 +1606,14 @@ def _ag_params(self) -> set:
     def _features(self):
         return self._features_internal
 
-    def _get_tags(self):
-        collected_tags = {}
+    def _get_tags(self) -> dict:
```

Comment:
discussion point: maybe call these `capabilities` vs `tags`? Tags seems too generic and not pointing to the main purpose of this labeling: what functionality is supported.

Reply:
this is the API used by sklearn for this logic, so I'd prefer to be consistent with them.